interface/client/mistAPIBackend.js

```diff
@@ -56,6 +56,13 @@ mistAPIBackend = function(event) {
         }});
     }
 
+    // SET APPBAR
+    if(event.channel === 'appBar') {
+        Tabs.update(template.data._id, {$set:{
+            appBar: arg
```

Comment:
to whatever value the user adds on their `ethereum-dapp-url-bar-style` meta tag.